components/Subscribe/IFTTTItem.js

```diff
@@ -0,0 +1,155 @@
+import { useState, forwardRef, memo } from 'react';
+import { t } from 'ttag';
+import Modal from '@material-ui/core/Modal';
+import Button from '@material-ui/core/Button';
+import TextField from '@material-ui/core/TextField';
+import Typography from '@material-ui/core/Typography';
+import Box from '@material-ui/core/Box';
+import CloseIcon from '@material-ui/icons/Close';
+import IconButton from '@material-ui/core/IconButton';
+
+import { makeStyles } from '@material-ui/core/styles';
+import { ButtonIcon } from './FeedDisplayComponents';
+
+import CopyButton from '../CopyButton';
+
+const useStyles = makeStyles(theme => ({
+  paper: {
+    position: 'absolute',
+    width: 730,
+    backgroundColor: theme.palette.background.paper,
+    borderRadius: 10,
+    boxShadow: theme.shadows[5],
+    padding: theme.spacing(3, 7),
+    top: `50%`,
+    left: `50%`,
+    transform: `translate(-50%, -50%)`,
+  },
+  close: {
+    position: 'absolute',
+    top: `2%`,
+    right: `2%`,
+  },
+  youtube: {
+    position: 'relative',
+    width: '100%',
+    paddingTop: '56.25%',
+    marginTop: 20,
+    marginBottom: 20,
+    '& .iframe': {
+      position: 'absolute',
+      top: '0',
+    },
+  },
+  table: {
+    width: '100%',
+    borderCollapse: 'collapse',
+    borderRadius: '4px',
+    borderStyle: 'hidden',
+    boxShadow: '0 0 0 1px #ddd',
+    marginTop: 20,
+    marginBottom: 20,
+    '& td': {
+      border: '1px solid #ddd',
+      textAlign: 'center',
+    },
+  },
+  textField: { padding: '0 14px' },
+}));
+
+const ModalBody = forwardRef(function ModalBody(props, ref) {
+  const { IFTTTAppletUrl, feedUrl, tutorialYoutubeId, platform, close } = props;
+  const classes = useStyles();
+
+  return (
+    <div ref={ref} className={classes.paper} tabIndex={-1}>
+      <IconButton className={classes.close} onClick={close}>
+        <CloseIcon />
+      </IconButton>
+      <Typography variant="h5">{t`Subscribe to ${platform}`}</Typography>
+      <div className={classes.youtube}>
+        <iframe
+          className="iframe"
+          width="100%"
+          height="100%"
+          src={`https://www.youtube.com/embed/${tutorialYoutubeId}`}
+          frameBorder="0"
+          allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
+          allowFullScreen
+        ></iframe>
+      </div>
+      <table className={classes.table}>
```

Comment:
It's not recommended to use `<table>` for something that is not a data table.
I suggest using flexbox for this particular UI, which should yield much simpler DOM and CSS style.

Reply:
agree